internal/planner/telemetry.go

```diff
@@ -19,10 +19,7 @@ import (
 	"go.opentelemetry.io/otel/metric"
 )
 
-const (
-	flavorPlatform           = "github" // Currently only github is supported
-	flavorPressureMetricName = "mayfly_planner_flavor_pressure"
-)
+const flavorPressureMetricName = "mayfly_planner_flavor_pressure"
```

Comment:
Sorry we overlooked this. In previous PR's we agreed to use github_runner instead of mayfly. Look e.g. at the `telemetry` module in the webhook-gateway, the metrics are called like `github-runner.webhook.gateway.inbound.errors`